### ISSUE TITLE
Setup Bug report Github

### ISSUE THREAD

**Issue by prk-dismoi** [dis-moi/mobile_poc#179]

**Comment by lutangar** [comment]
I'm really like what they did at __PeerTube__ :
https://github.com/Chocobozzz/PeerTube/issues/new/choose

This should help us figuring out some missing pieces... do we need to setup a chat, forum, discord? We don't have some informal communication chanell with our community, something more casual than Github or emails...

> See issue template implementation here, https://github.com/Chocobozzz/PeerTube/tree/develop/.github/ISSUE_TEMPLATE
> @prk-dismoi you should be able to do the same just from the Github interface :)
> ping me if you need some more guidance

**Comment by prk-dismoi** [comment]
Yep this is still on the list, was circling back to it. Makes a second topic for us to discuss on Wednesday am if you can join.

**Comment by prk-dismoi** [comment]
@felix-lambert as discussed you can add your test scenarii here: https://docs.google.com/spreadsheets/d/1R5GtiGijuhtQbAJZk8ym7y4pWDQb5LM5HjznIptMIBE/edit#gid=1756126598
Either in functional testing tab or a new one.

**Comment by felix-lambert** [comment]
I think it's also possible to add scenarios based on the "Done" section in the "Projects" dashboard
